install_config/registry/deploy_registry_existing_clusters.adoc

```diff
@@ -79,6 +79,9 @@ To see a full list of options that you can specify when creating the registry:
 ----
 $ oc adm registry --help
 ----
+
+Please note that the value for the `--fs-group` option should be agreed with
```

Comment:
/should be agreed with/must be permitted by the fsgroup setting of the SCC/ 

(which SCC is used?  restricted?  they need to know which one to look at)

Reply:
In general you don't know which one will be used.

If you use `oc cluster up`, the restricted SCC is applied and in this case you should choose a value from the range:
```
$ oc -n default get ns default -o template --template='{{index .metadata.annotations "openshift.io/sa.scc.supplemental-groups"}}{{"\n"}}'
1000000000/10000
```

Or you can create an SCC with the specific supplemental group and add the service account to it. And this SCC will be applied to the registry automagically if you create the deployment config with this group.

Reply:
ok, i'd just hand wave something like `the value for --fs-group must be permitted by the SCC used by the registry (typically, the restricted SCC)`